ProcessMaker/EncryptedData/Local.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace ProcessMaker\EncryptedData;
+
+use Illuminate\Support\Facades\App;
+use Illuminate\Encryption\Encrypter;
+use Illuminate\Support\Str;
+use ProcessMaker\EncryptedData\EncryptedDataInterface;
+use ProcessMaker\Models\EncryptedData;
+
+class Local implements EncryptedDataInterface
+{
+    const ENCRYPTION_METHOD = 'aes-256-cbc';
+
+    public static $iv = '';
+
+    /**
+     * Encrypt text.
+     *
+     * @param string $plainText
+     * @param string $iv
```

Comment:
Update the documentation

```suggestion
     * @param string|null $iv
```